src/openqaoa-core/backends/basebackend.py

```diff
@@ -25,20 +11,27 @@
 """
 from abc import ABC, abstractmethod, abstractproperty
 from typing import Union, List, Dict, Optional, Any, Tuple
+from copy import deepcopy
 import numpy as np
 
-from .devices import DeviceBase
-from .qaoa_parameters.gatemap import RotationGateMap, TwoQubitRotationGateMap
-from .qaoa_parameters.baseparams import QAOACircuitParams, QAOAVariationalBaseParams
-from .utilities import qaoa_probabilities, round_value
+from .devices_core import DeviceBase
+from ..qaoa_components import (
+    RotationGateMap,
+    TwoQubitRotationGateMap,
+    QAOADescriptor,
+    QAOAVariationalBaseParams,
+)
+from ..utilities import qaoa_probabilities, round_value
 from .cost_function import cost_function
 from copy import deepcopy
```

Comment:
Move it to line 16!

Reply:
Will be changed in a newer commit.

Reply:
Resolved in commit [27bae4a](https://github.com/entropicalabs/openqaoa/pull/170/commits/27bae4a996892f17dd47b786165d00be1c1b1d22)